helm/values.yaml

```diff
@@ -0,0 +1,76 @@
+# Default values for immudb.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: codenotary/immudb
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: ""
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+
+adminPassword: ""
+
+serviceAccount:
+  # Specifies whether a service account should be created
+  create: false
+  # Annotations to add to the service account
+  annotations: {}
+  # The name of the service account to use.
+  # If not set and create is true, a name is generated using the fullname template
+  name: ""
+
+podAnnotations: {}
+
+podSecurityContext: {}
+  # fsGroup: 2000
+
+securityContext: {}
+  # capabilities:
```

Comment:
Do you think we could enable this by default? The goal is to make sure immudb works with the least privileged setup we could find. If someone needs some more permission for some reason (maybe some additional initialization script) he could then provide his own `securityContext` value.

Reply:
We can enable it, but we must turn off `readOnlyRootFilesystem`, as we need to write to the disk. Done in commit 5828ab09

Reply:
Isn't is that only the data folder requires write permissions? If so then it should be mounted as a volume and be have read-write access even if the root fs is read-only

Reply:
Done in commit 725545aa